Fika.Core/UI/Patches/FikaVersionLabel_Patch.cs

```diff
@@ -15,9 +15,18 @@ public class FikaVersionLabel_Patch : ModulePatch
     {
         private static string versionLabel;
 
+        private static Traverse preloaderUiTraverse;
```

Comment:
Is there a specific reason for saving these as a variable in the memory constantly?
I'd prefer doing the `Traverse` when running the code, as there is really no reason to keep these saved in memory.

Reply:
solved, thks.